schema.rst

```diff
@@ -292,14 +290,14 @@ shall be explicitly undefined (e.g. ``-UFOO`` passed to the compiler).
 A map may be used instead to give different values
 depending on the language of the consuming source file.
 In this case, the build tool shall select the list
-from the map whose key matches the (case-insensitive) language
+from the map whose key matches the (case-sensitive) language
```

Comment:
Would it be better to include lower case here, `(lower-case, case-sensitive)`?

Reply:
Done.